core/variables.tf

```diff
@@ -41,3 +71,40 @@ variable "rds_password" {
   type        = string
   sensitive   = true
 }
+
+variable "bastion_instance_type" {
+  description = "The bastion instance type"
+  default     = "t3.nano"
+}
+
+variable "bastion_instance_desired_count" {
+  description = "The desired number of the bastion instance"
+  default     = 1
+}
+
+variable "bastion_max_instance_count" {
+  description = "The maximum number of the instance"
+  default     = 1
+}
+
+variable "bastion_min_instance_count" {
+  description = "The minimum number of the instance"
+  default     = 1
+}
+
+variable "bastion_ami_id" {
+  description = "The AMI ID for the instance"
+  default     = "ami-01811d4912b4ccb26"
+}
+
+variable "redis_host" {
```

Comment:
why is the Redis host variable? why is it not being retrieved from the ElastiCache module?

Reply:
Updated in [d59bff1](https://github.com/nvminhtue/tyson_devops_ic/pull/33/commits/d59bff1af0d01a550f845a8a9064cff68cf37903)
![image](https://github.com/user-attachments/assets/662b2444-9631-4f8a-ab51-4e647e54fa70)